### ISSUE TITLE
Bugs with hopoints when exporting from the editor to html

### ISSUE THREAD

**Issue by dzhakhangirov** [google/model-viewer#4087]
Hotpoints from the editor (https://modelviewer.dev/editor/) are exported with errors to html format. That's what I see in the editor (1 pic). And this is how I see it (2 pic) on the website (https://arguv.fun/test/model010223/). Please fix it.

#### Live Demo
https://arguv.fun/test/model010223/

### Browser Affected

- [ ] Chrome, version: 109.0.5414.120
- [ ] Edge
- [ ] Firefox

### OS

- [ ] Windows

![editor](https://user-images.githubusercontent.com/124194927/216135054-73f1906a-c838-44e1-9921-68c0ff14e261.png)
![site](https://user-images.githubusercontent.com/124194927/216135085-ed86e876-ab21-46a1-ad36-86bbc19ac82a.png)

**Comment by elalish** [comment]
Yes, this will be fixed by the new release, see https://github.com/google/model-viewer/discussions/4076#discussioncomment-4835092

**Comment by dzhakhangirov** [comment]
thanks
